opencti-platform/opencti-graphql/config/test.json

```diff
@@ -7,7 +7,7 @@
     "sync_direct_start_remote_uri": "http://localhost:4200/graphql",
     "sync_restore_start_remote_uri": "http://localhost:4200/graphql",
     "app_logs": {
-      "logs_level": "error",
+      "logs_level": "info",
```

Comment:
This changes is wanted ?

Reply:
yes: this allows to have more logs, specially with client python